Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+--
+
+CREATE SEQUENCE "XML"."TraML_ID_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE "XML"."TraML_ID_seq" OWNER TO postgres;
+
+--
+-- TOC entry 2974 (class 0 OID 0)
+-- Dependencies: 225
+-- Name: TraML_ID_seq; Type: SEQUENCE OWNED BY; Schema: XML; Owner: postgres
+--
+
+ALTER SEQUENCE "XML"."TraML_ID_seq" OWNED BY "XML"."TraML"."ID";
+
+
+--
+-- TOC entry 2785 (class 2604 OID 17696)
+-- Name: Metadata_Proteomics ID; Type: DEFAULT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Metadata_Proteomics" ALTER COLUMN "ID" SET DEFAULT nextval('"MzTab"."Metadata_Proteomics_Internal_ID_seq"'::regclass);
+
+
+--
+-- TOC entry 2787 (class 2604 OID 18183)
+-- Name: Peptide Section id; Type: DEFAULT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Peptide Section" ALTER COLUMN id SET DEFAULT nextval('"MzTab"."Peptide Section_id_seq"'::regclass);
+
+
+--
+-- TOC entry 2786 (class 2604 OID 18173)
+-- Name: Protein section id; Type: DEFAULT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Protein section" ALTER COLUMN id SET DEFAULT nextval('"MzTab"."Protein section_id_seq"'::regclass);
+
+
+--
+-- TOC entry 2783 (class 2604 OID 17709)
+-- Name: Metadata_metabolomics ID; Type: DEFAULT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Metadata_metabolomics" ALTER COLUMN "ID" SET DEFAULT nextval('"MzTab-M"."Metadata_metabolomics_Internal_mzTab-ID_seq"'::regclass);
+
+
+--
+-- TOC entry 2790 (class 2604 OID 18229)
+-- Name: Small Molecule Evidence (SME) Section SME_ID; Type: DEFAULT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Evidence (SME) Section" ALTER COLUMN "SME_ID" SET DEFAULT nextval('"MzTab-M"."Small Molecule Evidence (SME) Section_SME_ID_seq"'::regclass);
+
+
+--
+-- TOC entry 2791 (class 2604 OID 18240)
+-- Name: Small Molecule Feature (SMF) Section SMF_ID; Type: DEFAULT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Feature (SMF) Section" ALTER COLUMN "SMF_ID" SET DEFAULT nextval('"MzTab-M"."Small Molecule Feature (SMF) Section_SMF_ID_seq"'::regclass);
+
+
+--
+-- TOC entry 2784 (class 2604 OID 18215)
+-- Name: Small Molecule Section SML_ID; Type: DEFAULT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Section" ALTER COLUMN "SML_ID" SET DEFAULT nextval('"MzTab-M"."Small Molecule Section_SML_ID_seq"'::regclass);
+
+
+--
+-- TOC entry 2789 (class 2604 OID 18194)
+-- Name: FeatureXML ID; Type: DEFAULT; Schema: XML; Owner: postgres
+--
+
+ALTER TABLE ONLY "XML"."FeatureXML" ALTER COLUMN "ID" SET DEFAULT nextval('"XML"."FeatureXML_ID_seq"'::regclass);
+
+
+--
+-- TOC entry 2788 (class 2604 OID 18205)
+-- Name: TraML ID; Type: DEFAULT; Schema: XML; Owner: postgres
+--
+
+ALTER TABLE ONLY "XML"."TraML" ALTER COLUMN "ID" SET DEFAULT nextval('"XML"."TraML_ID_seq"'::regclass);
```

Comment:
Double check that the sequences defined here cover the above tables I mention individually in the comments.